Indicators/MovingAverageTypeExtensions.cs

```diff
@@ -35,18 +35,28 @@ public static IndicatorBase<IndicatorDataPoint> AsIndicator(this MovingAverageTy
             {
                 case MovingAverageType.Simple:
                     return new SimpleMovingAverage(period);
+
                 case MovingAverageType.Exponential:
                     return new ExponentialMovingAverage(period);
+
                 case MovingAverageType.Wilders:
                     return new ExponentialMovingAverage(period, 1m/period);
+
                 case MovingAverageType.LinearWeightedMovingAverage:
                     return new LinearWeightedMovingAverage(period);
+
                 case MovingAverageType.DoubleExponential:
                     return new DoubleExponentialMovingAverage(period);
+
                 case MovingAverageType.TripleExponential:
                     return new TripleExponentialMovingAverage(period);
+
                 case MovingAverageType.Triangular:
                     return new TriangularMovingAverage(period);
+
+                case MovingAverageType.T3:
+                    return new T3MovingAverage(period, 1);
```

Comment:
Looking online it seems the normal default value for the volume factor is 0.7? Did TA-lib use a value of 1?

http://www.binarytribune.com/forex-trading-indicators/t3-moving-average-indicator/
http://www.multicharts.com/support/base/?action=article&id=1419